lib/omnibus/packagers/msi.rb

```diff
@@ -47,24 +47,46 @@ class Packager::MSI < Packager::Base
       FileSyncer.glob("#{resources_path}/assets/*").each do |file|
         copy_file(file, "#{resources_dir}/assets/#{File.basename(file)}")
       end
+
+      # If fastmsi, copy externally built custom action DLL into the staging directory
+      copy_file("#{resources_path}/ext/CustomActionFastMsi.CA.dll", "#{staging_dir}") if fast_msi
```

Comment:
This DLL should ship as part of the Omnibus framework (in the gem's `resources/msi` directory). It can be copied into the staging directory like so:

```ruby
copy_file(resource_path('CustomActionFastMsi.CA.dll'), staging_dir)
```